sigs.yaml

```diff
@@ -42,35 +45,15 @@ sigs:
     day: Tuesday
     time: "10:00"
     tz: EST
-    frequency: Bi-Weekly
-    url: meet.google.com/zfj-woub-xwp
+    frequency: weekly
+    url: tbd
   contact:
-    slack: sig-operations
+    slack: operations
+    mailing_list: community@lists.operate-first.cloud
   subprojects:
-  - name: cluster-ops
```

Comment:
Why does this PR remove all the sig-operations subproject data? This change isn't mentioned in either the PR description or the comments.

Reply:
This change and the next one are not intentional. It appears that despite fetching and rebasing on upstream, I managed to have a sigs.yaml file that was not entirely what is in upstream

Upstream: https://github.com/operate-first/community/commits/main/sigs.yaml
Mine: https://github.com/quaid/community/commits/quaid-add-sig-community-to-toc/sigs.yaml

I'm not sure where it got trimmed out, if I could find that point I suppose I can rebase there and that would cascade down to this PR?

Alternately I can just add it in and move on. :)

Reply:
OK, it's the same except I removed a line break that appeared in the middle of a sentence. It appears I also have a mailing list identified, are we not using that field? I can remove it too.